src/pages/software/index.md

```diff
@@ -1,60 +1,61 @@
 ---
-templateKey: 'software-page'
-path: /software
+templateKey: software-page
 seo:
-  title: Container Virtualization - Open Source VM Software - Kata Containers
-  description: Kata Containers is Apache 2 licensed open source software consisting of six components - Agent, Runtime, Proxy, Shim, Kernel and packaging of QEMU 2.11.
-  url: https://katacontainers.io/software/
+  description: >-
+    Kata Containers is Apache 2 licensed open source software consisting of six
+    components - Agent, Runtime, Proxy, Shim, Kernel and packaging of QEMU 2.11.
   image: /img/android-chrome-192x192.png
-  twitterUsername: "@katacontainers"
+  title: Container Virtualization - Open Source VM Software - Kata Containers
+  twitterUsername: '@katacontainers'
+  url: 'https://katacontainers.io/software/'
 title: Software
 subTitle: Development activities in Kata Containers
-intro: 
-  text: >
-    Kata Containers is Apache 2 licensed software consisting of six components: Agent, Runtime, 
-    Proxy, Shim, Kernel and packaging of QEMU 2.11. It is designed to be architecture agnostic, 
-    run on multiple hypervisors and plug seamlessly into the containers ecosystem.
+intro:
   buttons:
-    - text: explore on github
-      link: https://github.com/kata-containers
+    - link: 'https://github.com/kata-containers/kata-containers/'
+      text: explore on github
+  text: >
+    Kata Containers is Apache 2 licensed software consisting of six components:
+    Agent, Runtime,  Proxy, Shim, Kernel and packaging of QEMU 2.11. It is
+    designed to be architecture agnostic,  run on multiple hypervisors and plug
+    seamlessly into the containers ecosystem.
```

Comment:
Can you remove proxy and shim from this list as they don't exist in Kata 2.x. Also, I wouldn't specify the version of QEMU as it is bound to change so leaving it out makes this page "future proof". Further, we also package clh and fc so I'd consider wording like this:

> Kata Containers is Apache 2 licensed software consisting of two main components:
> the Kata agent, and the Kata Containerd shim v2 runtime. It also packages a Linux kernel and versions of QEMU, Cloud Hypervisor and Firecracker hypervisors.